extras/ioc_hash_scan.sh

```diff
@@ -0,0 +1,189 @@
+    --parameters="command=hostname" \
+    --output text)
+  echo "$HOST" | grep --invert-match "session" | sed '/^$/d'
+}
+
+function portForward() {
+  AWS_SHARED_CREDENTIALS_FILE="$AWS_CREDENTIALS_FILE" \
+    aws --profile="$AWSPROF" --region="$AWS_REGION" \
+    ssm start-session --target="$i" \
+    --document=AWS-StartPortForwardingSession --parameters="localPortNumber=5555,portNumber=6666"
+}
+
+function installNC() {
+  if [[ "$OS" == "Debian" ]]; then
+    AWS_SHARED_CREDENTIALS_FILE="$AWS_CREDENTIALS_FILE" \
+      aws --profile="$AWSPROF" --region="$AWS_REGION" \
+      ssm start-session --target="$i" \
+      --document=AWS-StartInteractiveCommand \
+      --parameters="command='sudo apt-get install netcat -y'"
+  else
+    AWS_SHARED_CREDENTIALS_FILE="$AWS_CREDENTIALS_FILE" \
+      aws --profile="$AWSPROF" --region="$AWS_REGION" \
+      ssm start-session --target="$i" \
+      --document=AWS-StartInteractiveCommand \
+      --parameters="command='sudo yum install netcat -y'"
+  fi
+}
+
+function startListen() {
+  if [[ "$OS" == "Debian" ]]; then
+    AWS_SHARED_CREDENTIALS_FILE="$AWS_CREDENTIALS_FILE" \
+      aws --profile="$AWSPROF" --region="$AWS_REGION" \
+      ssm start-session --target="$i" \
+      --document=AWS-StartInteractiveCommand \
+      --parameters="command='cd ~/src/ioc_scan; nc -l -p 6666 | tar xzf -'"
+  else
+    AWS_SHARED_CREDENTIALS_FILE="$AWS_CREDENTIALS_FILE" \
+      aws --profile="$AWSPROF" --region="$AWS_REGION" \
+      ssm start-session --target="$i" \
+      --document=AWS-StartInteractiveCommand \
+      --parameters="command='cd ~/src/ioc_scan; nc -l 6666 | tar xzf -'"
+  fi
+}
+
+## MAIN SCRIPT
+echo "IOC Hash Scan - $today-$(date +%H:%M:%S)" > "$logfile"
+
+for i in "${serverList[@]}"; do
+  OS="$(getOSType)"
+  if [[ "$OS" != "Debian" && "$OS" != "Fedora" ]]; then
+    echo "Non-supported OS Type"
+    exit 1
+  fi
+
+  instanceName=$(getHost)
+
+  echo "Begining scan of Instance $i -- $instanceName" | tee -a "$logfile"
+
+  # Start Port Forwarding
+  echo "Begining port forwarding"
+  portForward &
+
+  # Create ~/src/ioc_scan directory on Instance
+  echo "Verifying ~/src/ioc_scan directory on $instanceName"
+  AWS_SHARED_CREDENTIALS_FILE="$AWS_CREDENTIALS_FILE" \
+    aws --profile="$AWSPROF" --region="$AWS_REGION" \
+    ssm start-session --target="$i" \
+    --document=AWS-StartInteractiveCommand \
+    --parameters="command='if [ ! -d ~/src/ioc_scan ]; then mkdir -p ~/src/ioc_scan; fi'"
+
+  #Install netcat and start listening on port 6666
+  echo "Verifying netcat on $instanceName"
+  installNC
+  echo "Begin listening on $instanceName"
+  startListen &
+
+  # Copy latest ioc_scanner.py to target instance
+  curdir=$(pwd)
+  cd "$pydir" || exit
+
+  echo "Upload lastest ioc_scanner.py to $instanceName"
+  tar czf - ./ioc_scanner.py | nc localhost 5555
+
+  cd "$curdir" || exit
+
+  # Run ioc_scanner.py on target instance
+  echo "Scan $instanceName for IOC Hashes"
```

Comment:
Remove unnecessary capitalization:
```suggestion
  echo "Scan $instanceName for IOC hashes"
```